tsconfig.json

```diff
@@ -1,14 +1,16 @@
 {
   "compilerOptions": {
     "outDir": "./dist/",
-    "module": "es6",
+    "module": "commonjs",
```

Comment:
Just to check - do sign-in and upload still work? IIRC, I had to add `"module": "es6"` to fix the loading of the AWS SDK in TypeScript.

Reply:
Oddly, it's now only working with commonjs and not with es6 but it looks like the app is still working so I think this is ok.